cmd/run.go

```diff
@@ -396,5 +400,20 @@ func init() {
 	RunCmd.Flags().IntVar(&appHealthThreshold, "app-health-threshold", 0, "Number of consecutive failures for the app to be considered unhealthy")
 	RunCmd.Flags().BoolVar(&enableAPILogging, "enable-api-logging", false, "Log API calls at INFO verbosity. Valid values are: true or false")
 	RunCmd.Flags().StringVar(&apiListenAddresses, "dapr-listen-addresses", "", "Comma separated list of IP addresses that sidecar will listen to")
+	RunCmd.Flags().StringVarP(&appsRunConfigFile, "apps-run-config", "f", "", "The path to the configuration file for the apps to run")
```

Comment:
what about `run-file` instead of `apps-run-config` to be intuitive compared to `-f`?